doc/sphinx/run.rst

```diff
@@ -171,157 +171,219 @@ The equilibrium constant of the reaction is then given as
-signs, and the equilibrium constant attains the inverse value. Further,
-note that the equilibrium constant :math:`K` in Eq. [Keq] is the
-dimensionless *thermodynamic* equilibrium constant :math:`K_\mathrm{p}`.
-Apparent, concentration-based equilibrium constants can also be found in
-literature. To be used as input for the reaction ensemble, they need to
-be converted to thermodynamic constants as described in texbooks of
-Physical Chemistry. As a special case, all stoichiometric coefficients
-on one side of Eq.[general-eq] can be zero. Such reaction is equivalent
-to exchange with a reservoir, and the simulation in reaction ensemble
-becomes equivalent with the grandcanonical simulation. A simulation in
-the reaction ensemble consists of two types of moves: the reaction move
-and the configuration move. The configuration move is carried out by a
+of the reaction, and :math:`\mu_i^{\ominus}` the standard chemical
+potential (per particle) of species :math:`i`. Note that thermodynamic equilibrium is
+independent of the direction in which we write the reaction. If it is
+written with left and righ-hand side swapped, 
+both :math:`\Delta_{\mathrm{r}}G^{\ominus}` and the stoichiometric
+coefficients attain opposite signs, and the equilibrium constant attains the inverse value. 
+Further, note that the equilibrium constant :math:`K` is the
+dimensionless *thermodynamic, concentration-based* equilibrium constant,
+defined as
+
+.. math::
+
+   K(c^{\ominus}) = (c^{\ominus})^{-\bar\nu} \prod_i (c_i)^{\nu_i}
+
+wher :math:`\bar\nu=\sum_i \nu_i`, and :math:`c^{\ominus}` is the reference concentration,
+at which the standard chemical potential :math:`\Delta_{\mathrm{r}}G^{\ominus}` was determined.
+In practice, this constant is often used with the dimension of :math:`(c^{\ominus})^{\bar\nu}`
+
+.. math::
+
+   K_c(c^{\ominus}) = K(c^{\ominus})\times (c^{\ominus})^{\bar\nu}
+
+A simulation in
+the reaction ensemble consists of two types of moves: the *reaction move*
+and the *configuration move*. The configuration move is performed by a
 suitable molecular dynamics or a Monte Carlo scheme. The
 ``reacton_ensemble`` command of takes care only of the reaction moves.
 In the *forward* reaction, the appropriate number of reactants (given by
 :math:`\nu_i`) is removed from the system, and the concomitant number of
-products is inserted into the system. In the *reverse* reaction,
+products is inserted into the system. In the *backward* reaction,
 reactants and products exchange their roles. The acceptance probability
 :math:`P^{\xi}` for move from state :math:`o` to :math:`n` reaction
-ensemble is given by the criterion ::cite:`smith94a`
+ensemble is given by the criterion :cite:`smith94a`
 
 .. math::
 
-   P^{\xi} = \text{min}\biggl(1,V^{\bar\nu\xi}\Gamma^{\xi}e^{-\beta\Delta E_\mathrm{pot}}\prod_{i=1}\frac{N_i^0!}{(N_i^0+\nu_{i}\xi)!}
+   P^{\xi} = \text{min}\biggl(1,V^{\bar\nu\xi}\Gamma^{\xi}e^{-\beta\Delta E}\prod_{i=1}\frac{N_i^0!}{(N_i^0+\nu_{i}\xi)!}
```

Comment:
I would rather leave \Delta E_\mathrm{pot] here. It makes the equation self explanatory without further needing to read the description of the symbols.